docs/Options.md

```diff
@@ -92,7 +92,7 @@ slack` after changing it to take effect.
 
 **Default:** ``
 
-**Description:** If set, file attachments will be automatically downloaded to this location.
+**Description:** If set, file attachments will be automatically downloaded to this location. "%h" will be replaced by WeeChat home, "~/.weechat" by default.
```

Comment:
This file is generated, so you need to change the description in [wee_slack.py](https://github.com/wee-slack/wee-slack/blob/7bff4f2856a2cc8126fa8d2fecbd07e0422f2fb4/wee_slack.py#L4344-L4347), and run `./generate_docs.py` instead of changing this file manually.